build_runner_core/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.0.1
```

Comment:
```suggestion
## 4.0.1-dev
```

Reply:
It's moot now since I rebased to `4.1.0-dev`

This has come up in the past - I thought we had landed on putting `-dev` in the pubspec but not bothering with the changelog...

Reply:
I like it in both myself, I would actually lean towards having it in the changelog and not the pubspec if anything?

Reply:
I mostly just wanted an excuse to use the suggestion feature since @evanweible-wf enlightened me to its presence.

Reply:
> I would actually lean towards having it in the changelog and not the pubspec if anything?

It's easier to forget to touch it if it's only in the changelog.

Reply:
I disagree, you can't publish a version that is already published so you are physically incapable of not updating the pubspec. The changelog on the other hand is easy to forget.

Reply:
I still prefer both though, I like them being in sync.